inspire/modules/migrator/tasks.py

```diff
@@ -124,10 +125,12 @@ def migrate_chunk(chunk, broken_output=None, dry_run=False):
                 before_record_index.send(recid, json=json)
                 json.update({'_index': index, '_type': 'record', '_id': recid})
                 records_to_index.append(json)
-            except Exception:
+            except Exception as err:
+                current_app.logger.exception(err)
```

Comment:
Down below in `create_record` I was using: 
```python
logger.exception("Error in elaborating record ID {}".format(recid))
```

this should use the native Celery logging. What would be best (it passes automatically the exception)